azure-pipelines.yml

```diff
@@ -0,0 +1,49 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  # batch runs this pipeline one after the other (useful when there are many contributors)
+  batch: true
+  branches:
+    include:
+      - master
+      - develop
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - master
+      - develop
+
+pool:
+  vmImage: ubuntu-latest
+
+stages:
+- stage: Initial
+  jobs:
+  - job: Initializing
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '14.17.x' 
+      #force32bit: false # Optional
+      #checkLatest: false # Optional #version LTS o latest de node
+
+    # installing dependencies (react, axios, etc)
+    - script: npm install
+
+    # building the webApp
+    - script: npm run build
+    #for conditional run of a step or job or stage
+
+    
+    - task: PublishBuildArtifacts@1
```

Comment:
indentation is important for markdown readers so let's keep a 2 spaces indentation